tsconfig.json

```diff
@@ -17,5 +17,8 @@
       "*": ["node_modules/*", "src/types/*"]
     },
     "lib": ["es2017", "dom"]
+  },
+  "angularCompilerOptions": {
+    "enableIvy": true
```

Comment:
Why did you enable ivy?

Reply:
so angular devtools work: https://chrome.google.com/webstore/detail/angular-devtools/ienfalfjdbdpebioblfackkekamfmbnh

Reply:
👍